### PR TITLE
RHD-282 Activated footer social links

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

File: stylesheets/_global-wrapper.scss [hunk 1]
```diff
@@ -185,23 +185,27 @@ ul.footer-nav {
   }
 }
 
-ul.social-nav {
+ul#social-nav {
```
Comment (wesbos) [inline stylesheets/_global-wrapper.scss]: We try to stay away from using IDs to style elements in our CSS - can you switch this back to a class?

IDs can only be used a single time on the page, by using classes it allows us to reuse elements if we need to.

File: stylesheets/_global-wrapper.scss [hunk 2]
```diff
@@ -185,23 +185,27 @@ ul.footer-nav {
...
   list-style: none;
-  width:65%;
-  float: left;
   margin: 0;
-  text-align:right;
-  padding-right:5%;
+  margin-left: 5%;
+  padding: .3rem 0;
```
Comment (wesbos) [inline stylesheets/_global-wrapper.scss]: We're using PX on this site unless a module entirely uses `rems` - it's ideal not to mix the two back and forth.

If you must use rems, use the `remCalc(20)` mixin to convert 20px to rems

File: stylesheets/_global-wrapper.scss [hunk 3]
```diff
@@ -185,23 +185,27 @@ ul.footer-nav {
...
+    margin-left: 16px;
     float:none;
     a {
-      font-size: 25px;
+      font-size: 1.1rem;
     }
   }
 }
+@include tablet-landscape-and-down {
+  ul#social-nav {
```
Comment (wesbos) [inline stylesheets/_global-wrapper.scss]: Again - classes please

**Comment by wesbos** [comment]
A couple changes needed and then this is good to merge.

**Comment by adelasofia** [comment]
@wesbos Thanks for the feedback, I just merged in those fixes

**Comment by wesbos** [comment]
:+1: Good to merge

**Pull request merged**